sympy/functions/elementary/hyperbolic.py

```diff
@@ -273,6 +277,48 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    def _eval_is_positive(self):
+        arg = self.args[0]
+
+        if arg.is_real:
+            return True
+
+        re, im = arg.as_real_imag()
+        im_mod = im % (2*pi)
+
+        if im_mod == 0:
+            return True
+
+        if re == 0:
+            if im_mod < pi/2 or im_mod > 3*pi/2:
+                return True
+            elif im_mod >= pi/2 or im_mod <= 3*pi/2:
+                return False
+
+        return Or(And(Eq(re, 0), Or(im_mod < pi/2, im_mod > 3*pi/2)), Eq(im_mod, 0))
```

Comment:
The `_eval_is` routines should should return non-symbolic fuzzy bool (True/False/None) rather than symbolic Boolean (S.True/S.False/Boolean).

We could replace the logic above with fuzzy_or, fuzzy_and, re.is_zero etc. At the end if no determination is possible we should always return None (although fuzzy_or can do that for you).